test/consensus_cogateway/prove_consensus_gateway_proof.json

```diff
@@ -0,0 +1,17 @@
+  "rlpAccountNode": "0xf8440180a0ee8b5a291934299fb1b67ab9677a2b19b426c82dad727f7def6e86f42a9fcaa0a044134190597c53773b15aec278f3fda766d001d0ee06021839b4018d7e3f6a41"
+}
```

Comment:
Since the proof files are increasing, we should create test/data directory and put all the proof data files there. We followed similar approach in mosaic-contracts.
https://github.com/mosaicdao/mosaic-contracts/tree/develop/test/data

Reply:
Done